### ISSUE TITLE
Provide source branch name for PRs

### ISSUE THREAD

**Issue by zarbis** [jenkinsci/multibranch-action-triggers-plugin#14]
**Is your feature request related to a problem? Please describe.**
I'm using this plugin to cleanup test environments. To be precise, on branch delete I'm running this command:
```
helmDelete(env.SOURCE_PROJECT_FULL_NAME.replaceAll('/','-').replaceAll('%2F','-'))
```
I'm naming test environments like `$org-$repo-$branch`, e.g. `foo-bar-feature-baz` and this plugin works great for branches that are merged directly.
However, when branches go through PR and code review process `SOURCE_PROJECT_FULL_NAME` looks like `foo/bar/PR-123` and command above fails due to name mismatch.

**Describe the solution you'd like**
Having separate variable with source (and possibly target) branch name(s) would do the trick.

**Comment by techzio-io** [comment]
Hi, 
If these values are provided by the PR, they will be passed to the jobs as parameter. I will start working on it

**Comment by aytuncbeken** [comment]
Hi, 
This request make sense. I will start developing.
If these value are provided (not null), they will be passed to the jobs.

**Comment by aytuncbeken** [comment]
HI @zarbis 

This feature is released with 1.8.6 version. Please check the documentation Section 6 for the usage.
Best
